common/source/docs/common-escs-and-motors.rst

```diff
@@ -11,14 +11,14 @@ ArduPilot supports a wide variety of ESCs and motors.  The pages below provide s
 .. toctree::
     :maxdepth: 1
 
-    Brushless ESC Related Configuration <common-dshot>
 [site wiki="copter,rover"]
     Brushed motors <common-brushed-motors>
 [/site]
 [site wiki="copter"]
     Booster motor <booster-motor>
 [/site]
-    Brushless ESCs <common-brushless-escs>
+    Brushless ESCs without BLHeli<common-brushless-escs>
```

Comment:
"Brushless ESCs" is probably not quite correct.  It is really "PWM ESCs" right?  I think it might be best to just list the types of ESC protocols covered which could be "PWM, OneShot, DShot"

The section below could just be "BLHeli ESCs"

Reply:
change to PWM,OneShot,DShot ESCs and BLHeli-S/BLHeli32 Escs